models/lpjguess/tests/Rcheck_reference.log

```diff
@@ -45,30 +45,7 @@ Strong dependencies not in mainstream repositories:
 * checking S3 generic/method consistency ... OK
 * checking replacement functions ... OK
 * checking foreign function calls ... OK
-* checking R code for possible problems ... NOTE
-getClass_Fluxes: no visible binding for global variable ‘zz’
-getClass_Individual: no visible binding for global variable ‘zz’
-getClass_Patchpft: no visible binding for global variable ‘npft’
-getClass_Patchpft: no visible binding for global variable ‘zz’
-getClass_Soil: no visible binding for global variable ‘zz’
-getClass_Sompool: no visible binding for global variable ‘zz’
-getClass_SompoolCent: no visible binding for global variable ‘zz’
-getClass_Vegetation: no visible binding for global variable ‘zz’
-model2netcdf.LPJGUESS: no visible binding for global variable
-  ‘read.table’
-write.insfile.LPJGUESS: no visible binding for global variable
-  ‘lpjguess_param_list’
-write.insfile.LPJGUESS: no visible global function definition for
-  ‘data’
-write.insfile.LPJGUESS: no visible binding for global variable
-  ‘co2.1850.2020’
-write.insfile.LPJGUESS: no visible global function definition for
-  ‘write.table’
-Undefined global functions or variables:
-  co2.1850.2020 data lpjguess_param_list npft read.table write.table zz
-Consider adding
-  importFrom("utils", "data", "read.table", "write.table")
-to your NAMESPACE file.
+* checking R code for possible problems ... OK
 
```

Comment:
Sorry, I think I botched the line numbers 🤦 -- you can delete these next four lines as well (49-53; from this empty line down through "See section ..."), so that what's  left after deletion is 
```
* checking R code for possible problems ... OK
* checking Rd files ... OK

Reply:
(This is me giving GitHub a grumpy look for making it so hard to comment on blocks of lines that aren't yet marked as changed)